lec20-publishing.Rmd

```diff
@@ -1,4 +1,261 @@
+
+Journal impact factors are a measure of the average number of citations articles
+from that journal have received over a period of a year. How we value science
+and scientists has been dominated in the past by the impact factors of the
+journals they publish their work as a proxy for the quality of their papers.
+
+Questions:
+
+- Is this the right way for us to be determining the quality of research?
+- And if not, what is the right way to value science and in turn, scientists
+behind the research?
+
+You can read [this article](http://www.nature.com/news/beat-it-impact-factor-publishing-elite-turns-against-controversial-metric-1.20224)
+in Nature about pushback against impact factors from influential players in
+scientific publishing.
+
+#### Preprint Servers
+
+While the definition may vary, for our purposes, we will define a preprint as
+a research article, review or body of scientific work that has yet to be published in a journal.
+
+- Do you have any hesitations about submitting to a preprint server?
+
+### Video: Open Access Explained! (8:23)
+
+<iframe width="420" height="315" src="https://www.youtube.com/embed/L5rVH1KGBCY" frameborder="0" gesture="media" allowfullscreen></iframe>
+Fake or non-existent impact factors.
+
+### The ethics and risks in sharing data
+
+‘Scooping’ is often cited as a concern when discussing sharing data. However,
+getting scooped from a published data source has been [empirically demonstrated](http://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002432)
+to be a rare phenomenon, but even some open advocates have had trouble supporting
+[blanket open data policies](https://emckiernan.wordpress.com/2014/02/26/my-concerns-about-ploss-new-open-data-policy/).
+
+Let's discuss the following:
+
+- Which concerns about data/code sharing do you feel have merit?
+- How can we mitigate these issues?
+
+## Submitting a Scientific Publication
+
+Now that we have a good idea of authorship and the scientific publication process,
+let's consider the steps you would take to actually submit a scientific item for
+publishing.
+
+### General Format
+
+Typically, a research article will contain the following sections:
+
+- ”Abstract”
+- “Background and Rationale”
+- “Research Objectives and Hypotheses”
+- “Methods”
+- “Results”
+- “Discussion”
+- “Conclusion”
+
+Do these look familiar? They should, because they are the headings you need to
+include in your final assignment! "Background and Rationale" and "Research
+Objectives and Hypotheses" are usually combined into one "Introduction" section,
+but this varies from journal to journal. Also, the order may vary with some
+placing the detailed methods in the supplementary. Journals may also require
+a short "Significance Statement" as to why the finding is interesting. This may
+be more common in basic science that has clinical/translational implications.
+Also, many journals also require a written abstract for their work as well.
+
+### Journal Specifications
```

Comment:
This is useful to include in the material, but you may not need to talk about this in much detail, if at all.